app/config/security.yml

```diff
@@ -60,7 +60,9 @@ security:
         # /admin accessible only from ADMIN, super users and AD users
         - { path: ^/admin, roles: [ROLE_ADMIN, ROLE_AD] }
         - { path: ^/ad, roles: [ROLE_AD] }
-        - { path: ^/pa, roles: [ROLE_PA, ROLE_PA_ADMIN, ROLE_PA_TEAM_MEMBER] }
+        - { path: /pa/$, roles: [ROLE_PA, ROLE_PA_ADMIN, ROLE_PA_TEAM_MEMBER] }
```

Comment:
http://symfony.com/doc/current/security/access_control.html
" the first rule that matches is used"
So, I think you could keep /pa last to be accessible from any PA, and move your more strictly rules around edit/add/delete before that